### DISCUSSION TITLE
loading theme from other domain fails with CORS error

### DISCUSSION THREAD

**Discussion by trusktr** [heremaps/harp.gl#2368] [Q&A]
When I try

```js
new MapView({
  theme: 'https://www.harp.gl/docs/master/examples/resources/berlin_tilezen_effects_outlines.json',
})
```

I get a familiar CORS error in the app:

```
MapView: Failed to load theme from https://www.harp.gl/docs/master/examples/resources/berlin_tilezen_effects_outlines.json: TypeError: Failed to fetch TypeError: Failed to fetch
```

How can we make this work?

**Comment by zbigg** [comment]
I didn't try for long time, but you may `@here/harp-map-theme` hosted directly from unpkg.com, try this: 

`https://unpkg.com/@here/harp-map-theme@0.28.0/resources/berlin_tilezen_effects_outlines.json`

Same url with browser for available themes: https://unpkg.com/browse/@here/harp-map-theme@0.28.0

At least, unpkg shouldn't emit CORS failures.
